go.mod

```diff
@@ -5,7 +5,8 @@ go 1.15
 require (
 	contrib.go.opencensus.io/exporter/zipkin v0.1.2
 	github.com/gin-gonic/gin v1.7.2
-	github.com/go-logr/stdr v1.2.2 // indirect
+	github.com/go-logr/logr v1.2.1
```

Comment:
@davexroth the only way get to get the compiler to complain and require these 2 is if you modify the code to define your own logger. Also there's no way to configure the logger using a file or env vars.